Dockerfile.pachctl

```diff
@@ -1,7 +1,6 @@
 # syntax=docker/dockerfile:1.0-experimental
 ARG GO_VERSION
-FROM golang:${GO_VERSION}
-RUN apt update && apt install ca-certificates
```

Comment:
I'd actually recommend leaving this line in, and copying the certificates to the final image.  

`COPY --from=pachctl_build /etc/ssl/certs /etc/ssl/certs`
`COPY --from=pachctl_build /usr/share/ca-certificates /usr/share/ca-certificates`

I think that's sufficient on debian to copy over the certs.  By default debian:buster-slim actually doesn't have any ca-certs installed.